openapi.yml

```diff
@@ -24,40 +24,7 @@ tags:
     description: Digital Repository Objects
   - name: files
     description: upload binary files
-  - name: monitoring
-    description: Monitoring and health checks
 paths:
-  /status:
```

Comment:
Isn't this an API though?

Reply:
Sure, in some sense it is. But this is overhead that buys us nothing. Having to document OKComputer and Sidekiq and whatever else is a hassle and makes the openapi.yml even more verbose.

If there is a specific desire to expose status as part of the API, it should be given a `/v1/status` endpoint, as it is in dor-services-app.

Reply:
Okay, that sounds reasonable.